app/src/main/res/layout/fragment_create_shop.xml

```diff
@@ -0,0 +1,251 @@
+                        android:layout_gravity="center"
+                        android:layout_marginStart="@dimen/dp_20"
+                        android:layout_marginTop="@dimen/dp_5"
+                        android:spinnerMode="dropdown"
+                        />
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="@dimen/dp_10"
+                        android:text="@string/two_dot"
+                        />
+                    <android.support.v7.widget.AppCompatSpinner
+                        android:layout_width="@dimen/dp_70"
+                        android:layout_height="@dimen/dp_40"
+                        android:layout_gravity="center"
+                        android:layout_marginTop="@dimen/dp_5"
+                        android:spinnerMode="dropdown"
+                        />
+                </LinearLayout>
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    >
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="@dimen/dp_2"
+                        android:layout_marginTop="@dimen/dp_10"
+                        android:paddingTop="@dimen/dp_10"
+                        android:text="@string/open_forever"
+                        android:textColor="@color/color_black"
+                        />
+                    <CheckBox
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="@dimen/dp_10"
+                        android:layout_marginTop="@dimen/dp_5"
+                        />
+                </LinearLayout>
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    >
+                    <TextView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="@dimen/dp_2"
+                        android:layout_marginTop="@dimen/dp_10"
+                        android:paddingTop="@dimen/dp_10"
+                        android:text="@string/auto_reject_time"
+                        android:textColor="@color/color_black"
+                        />
+                    <android.support.v7.widget.AppCompatSpinner
+                        android:layout_width="@dimen/dp_70"
+                        android:layout_height="@dimen/dp_40"
+                        android:layout_gravity="center"
+                        android:layout_marginStart="@dimen/dp_20"
```

Comment:
ERROR: [RtlCompat] To support older versions than API 17 (project specifies 14) you should *also* add `android:layout_marginLeft="@dimen/dp_20"`
 API 17 adds a `textAlignment` attribute to specify text alignment. However, if you are supporting older versions than API 17, you must *also* specify a gravity or layout_gravity attribute, since older platforms will ignore the `textAlignment` attribute.